tests/integrations/crossed_integrations/test_kafka.py

```diff
@@ -199,42 +199,10 @@ def validate_kafka_spans(self, producer_interface, consumer_interface, topic):
         return producer_span, consumer_span
 
 
-@scenarios.crossed_tracing_libraries
-@coverage.basic
-@features.kafkaspan_creationcontext_propagation_with_dd_trace_js
-class Test_NodeJSKafka(_Test_Kafka):
-    buddy_interface = interfaces.nodejs_buddy
-    buddy = _nodejs_buddy
-    WEBLOG_TO_BUDDY_TOPIC = "Test_NodeJSKafka_weblog_to_buddy"
-    BUDDY_TO_WEBLOG_TOPIC = "Test_NodeJSKafka_buddy_to_weblog"
-
-    @missing_feature(library="golang", reason="Expected to fail, Golang does not propagate context")
-    @missing_feature(library="ruby", reason="Expected to fail, Ruby does not propagate context")
-    @missing_feature(library="python", reason="Expected to fail, Python does not propagate context")
-    def test_produce_trace_equality(self):
-        super().test_produce_trace_equality()
-
-    @missing_feature(library="golang", reason="Expected to fail, Golang does not propagate context")
-    @missing_feature(library="ruby", reason="Expected to fail, Ruby does not propagate context")
-    @missing_feature(library="python", reason="Expected to fail, Python does not propagate context")
-    def test_consume_trace_equality(self):
-        super().test_consume_trace_equality()
-
-
-@scenarios.crossed_tracing_libraries
-@coverage.basic
-@features.kafkaspan_creationcontext_propagation_with_dd_trace_py
-class Test_PythonKafka(_Test_Kafka):
-    buddy_interface = interfaces.python_buddy
-    buddy = _python_buddy
-    WEBLOG_TO_BUDDY_TOPIC = "Test_PythonKafka_weblog_to_buddy"
-    BUDDY_TO_WEBLOG_TOPIC = "Test_PythonKafka_buddy_to_weblog"
-
-
 @scenarios.crossed_tracing_libraries
 @coverage.basic
 @features.kafkaspan_creationcontext_propagation_with_dd_trace_java
-class Test_JavaKafka(_Test_Kafka):
+class Test_Kafka(_Test_Kafka):
     buddy_interface = interfaces.java_buddy
     buddy = _java_buddy
     WEBLOG_TO_BUDDY_TOPIC = "Test_JavaKafka_weblog_to_buddy"
```

Comment:
```suggestion
    WEBLOG_TO_BUDDY_TOPIC = "Test_Kafka_weblog_to_buddy"
```

Probably want to change this here, since it's not about Java anymore?

Also, what about

```
    buddy_interface = interfaces.java_buddy
    buddy = _java_buddy
```

I suspect that code in `interfaces.java_buddy` needs further refactor or cleaning beyond a rename.

Reply:
Same for the feature name, `@features.kafkaspan_creationcontext_propagation_with_dd_trace_java`

Reply:
I've made the topic name change, as well as changing the feature name to `kafkaspan_creationcontext_propagation_with_dd_trace`. There isn't any code to switch in the java buddy, since the code updating the weblog was simply refactoring the Kafka endpoint, which is still being used. Same for the other buddies as well.